scripts/core/INTRO.rst

```diff
@@ -145,6 +145,18 @@ and followed by a digit or uppercase letter are reserved for use by the implemen
 Applications which use Unified Runtime must not provide definitions of these symbols.
 This allows the Unified Runtime shared library to be updated with additional symbols for new API versions or extensions without causing symbol conflicts with existing applications.
 
+Printing API
+------------
+## --validate=off
+The header "${x}_print.hpp" contains the "${x}_print" namespace with the output stream operator (<<) overloads. These are helpful for a human-friendly print
```

Comment:
`The header "${x}_print.hpp" contains the "${x}_print" namespace with the output stream operator (<<) overloads. These are helpful for printing Unified Runtime types in a human-friendly way`
?

Reply:
Rephrased in the #875 PR.